x-pack/plugins/apm/server/lib/alerts/register_transaction_error_rate_alert_type.ts

```diff
@@ -170,7 +172,11 @@ export function registerTransactionErrorRateAlertType({
             transactionType,
             environment,
             threshold: alertParams.threshold,
-            triggerValue: transactionErrorRate,
+            triggerValue: String(
+              parseFloat(transactionErrorRate.toPrecision(3))
+            ),
```

Comment:
This looks like some custom number formatting. Don't we have an existing formatter?

Reply:
I could use `asInteger` that converts this `3.374329704990765` into `3`, WDYT?

Reply:
Perhaps `asDecimalOrInteger` is better:

https://github.com/elastic/kibana/blob/ce97982522cfb2b1e6b69810e72310d30ba00120/x-pack/plugins/apm/public/utils/formatters/duration.ts#L34-L40

Btw. can you move that function so it's co-located with `asInteger`? And add a test for it 🙏

Reply:
yeah, that makes sense!

Reply:
done https://github.com/elastic/kibana/pull/78930/commits/e62cae24dfeec8e36e3e2d46b2c50e728dfac3b0